tests/FunctionsTest.php

```diff
@@ -185,6 +191,65 @@ public static function captureLastErrorDataProvider(): \Generator
         ];
     }
 
+    /**
+     * @dataProvider captureCheckInProvider
+     */
+    public function testCaptureCheckIn(array $expectedFunctionCallArgs)
+    {
+        $hub = new Hub();
+        $options = new Options([
+            'environment' => Event::DEFAULT_ENVIRONMENT,
+            'release' => '1.1.8',
+        ]);
+        /** @var ClientInterface&MockObject $client */
+        $client = $this->createMock(ClientInterface::class);
+        $client->expects($this->once())
+            ->method('getOptions')
+            ->willReturn($options);
+
+        SentrySdk::setCurrentHub($hub);
```

Comment:
We should avoid calls like this in the tests as it affects the singleton instance of the SDK, which is shared across all tests. While it's true that we have a PHPUnit listener that resets the state after a test ran, it's better to proactively avoid issues. Instead, if possible, prefer mocking the hub like we did in similar tests

Reply:
This is fine, we do this in other tests as well. We can fix it once it becomes a real issue.